modules/experimental/pages/cc-blast.adoc

```diff
@@ -0,0 +1,22 @@
+= CC Blast
+
+`Detachment ability`
+
+Some forces are able to disrupt their close combat opponents, for example by infecting them with Nurgle's Rot.
```

Comment:
I have some thoughts on how to edit this and the equivalent under _FF Blast_. Ideally, they'd mirror each other pretty closely in terms of the information structures that they map to.

However, the examples you've roughed out don't seem to parallel each other — which is partly a function of the fact that at present we don't parallel how we grant the effects in the builder, and partly a function of my unfamiliarity with fluff behind the term 'Kakophoni'.

At present, the builder grants _CC Blast_ through the Mark of Nurgle, whilst it grants _FF Blast_ not through the Mark of Slaanesh but rather through this other thing called 'Kakophoni'.

So, first: Will you bundle _FF Blast_ into the Mark of Slaanesh to mirror _CC Blast_ under Mark of Nurgle? Or, do you mean to use Mark of Slaanesh and Kakophoni in such a way that you can't do this because they don't always apply *together*?

Second: Is 'Kakophoni' a catch-all for various Slaanesh gifts like 'Allure of Slaanesh' (and therefore functionally equivalent to something 'Contagions' at the level of Mark of Nurgle)? Or is it a specific gift that applies to the likes of Noise Marines, as the name implies (and therefore functionally equivalent to something like Nurgle's Rot, at the level of the specific unit)?

Reply:
To summarise a long discussion on FB about Marks and rules:

Close combat disruption works for the Mark of Nurgle, not matter what type of detachment it's added to.
Ranged disruption isn't as good a fit for the Mark of Slaanesh, both from a background point of view and how it would help different detachments. In particular, all the original Slaaneshi Daemon units have 0 Firepower, and absolutely do _not_ want to be involved in a firefight!

For the Emperor's Children Chapter, on the other hand, firefight disruption is a perfect fit, to represent their sonic weapons, and is a much less drastic change than giving them all Disrupt. In modern background, the Kakophoni were originally a Heresy-era Emperor's Children unit that were the first to use sonic weaponry, and the term spread to cover Noise Marines and sonic weapons in general.

So the Mark of Slaanesh is Stubborn, representing the units not caring about Blast markers, and echoing the original Epic 40k Noise Marine unit having Stubborn.

But the Emperor's Children detachment _also_ has Kakophoni/FF Blast, and is the only detachment that has it so far. Although I'd like to add it to the Slaaneshi Scout Titans, and to the Supreme Commander detachment if a Noise Marine Commander is taken.

Reply:
Ah, gotcha — makes sense, thanks. I think I know what to do now. I'll add suggestions to both abilities shortly.

Reply:
```suggestion
These units have some special means to disrupt their opponents in close combat. 

For example, Plague Marines of Nurgle carry the potent daemonic disease known as Nurgle's Rot, which corrupts and debilitates those who come into contact with it.
```